python_scripts/ensemble_bagging.py

```diff
@@ -344,7 +335,7 @@ def bootstrap_sample(data, target):
 
 # %%
 for i, regressor in enumerate(bagging.estimators_):
-    regressor_predictions = regressor.predict(data_test)
+    regressor_predictions = regressor.predict(data_test.to_numpy())
```

Comment:
```suggestion
    # we convert `data_test` into a NumPy array to avoid a warning raised in scikit-learn
    regressor_predictions = regressor.predict(data_test.to_numpy())
```